kandinsky/src/color.cpp

```diff
@@ -7,22 +7,21 @@ KDColor KDColor::blend(KDColor first, KDColor second, uint8_t alpha) {
    * dealt with. So let's make a special case for them. */
   if (alpha == 0) {
     return second;
-  } else if (alpha == 0xFF) {
+  }
+  if (alpha == 0xFF) {
+    return first;
+  }
+  if (first == second) {
```

Comment:
IIRC, those 2 cases were supposed to simply be a fast path. Is it still the case with that new 3rd one? In other words, would the new case (`first == second`) yield correct results even if this test was disabled?

Reply:
With the change made to `oneMinusAlpha`, the case `first == second` is indeed nothing more than a shortcut. I did some benchmarking, and it seems to happen often enough for this check to yield some speed gain (albeit small when compared to the other two). I'll change the first comment to reflect that.